### DISCUSSION TITLE
Direct connection between smart watch and BLE device

### DISCUSSION THREAD

**Discussion by nubilion** [zepp-health/zeppos-forum#120] [Developer Feedback]
Hello, could you please confirm whether it is possible to use Bluetooth to connect smartwatch with some BLE device and exchange data between them. Currently the focus was mostly on communication between smart watch and app-side component on the phone but it is not clear whether smart watch can connect to some other BLE device beside smartphone.

**Comment by zdunex25** [comment]
Not possible in OS 2.0, I read somewhere it will be possible in os 3.0.

**Comment by Xuemuyang** [comment]
Hello, Zepp OS 3.0 will be open, stay tuned!
